analytics/analytics/core/models.py

```diff
@@ -72,6 +72,15 @@ class JobAttempt(models.Model):
     error_taxonomy = models.CharField(max_length=64)
 
 
+class SectionTimer(models.Model):
+    """
+    A model for storing the timer values for each section in GitLab.
+    """
+
+    job_attempt = models.OneToOneField(JobAttempt, on_delete=models.CASCADE)
+    timers = models.JSONField(default=dict)
+
+
```

Comment:
Would it make more sense to store this as a possibly nullable `timers` field on `JobAttempt`? Or is there a more concrete reason to have this as a separate model?